### PR TITLE
Subscriptions push doc adding

### PR DESCRIPTION
**Pull request by pjasiakcf**

### CODE REVIEW

File: APIs/Subscriptions.qmd [hunk 1]
```diff
@@ -0,0 +1,477 @@
+---
+title: Subscriptions 
+---
+# Push subscriptions
```
Comment (Pratichhya) [inline APIs/Subscriptions.qmd:4]: If you write the title in the yml at the top, then it is suggested not to again include the title below in the markdown, i.e. # Push subscriptions. If needed, you can make it heading 2 with ##

Comment (pjasiakcf) [reply APIs/Subscriptions.qmd]: @Pratichhya I've updated it within the document

Comment (Slegersj) [reply APIs/Subscriptions.qmd]: @pjasiakcf Hi Paulina, I am trying to fix the Subscription Entity description table output using: https://quarto.org/docs/authoring/tables.html

Comment (pjasiakcf) [reply APIs/Subscriptions.qmd]: @Slegersj thank you for your help with the table. I see that there is still an issue with formatting within the table and that words are splitted in a column. Can we try to fix it somehow? We can also try to change the width of the columns.

Comment (Slegersj) [reply APIs/Subscriptions.qmd]: @pjasiakcf I will check as soon as possible if the table settings used [here](https://documentation.dataspace.copernicus.eu/Quotas.html#copernicus-general-users) can help us.

Comment (pjasiakcf) [reply APIs/Subscriptions.qmd]: Hi @Slegersj, now I see that we have the conflict in the branch and the preview still shows the wrong ver. of the table.

Comment (Slegersj) [reply APIs/Subscriptions.qmd]: Hi @pjasiakcf I resolved the conflict in the branch and you can check the new formatting of the table.

Comment (pjasiakcf) [reply APIs/Subscriptions.qmd]: Hi @Slegersj  still I see the words splitted in the first column. Is it possible to fix? If not I think we can publish it as it is. It looks much better now.

**Pull request merged**